Userland/Libraries/LibGUI/CMakeLists.txt

```diff
@@ -87,6 +87,7 @@ set(SOURCES
     SeparatorWidget.cpp
     SettingsWindow.cpp
     Shortcut.cpp
+    ShutdownDialog.cpp
```

Comment:
`ShutdownDialog` is not appropriate for LibGUI, it's not a generally useful GUI class.

Reply:
Makes sense. How about leaving it in Taskbar and doing the following:
```patch
diff --git a/Userland/Services/LoginServer/CMakeLists.txt b/Userland/Services/LoginServer/CMakeLists.txt
index e341c88974..26689fff64 100644
--- a/Userland/Services/LoginServer/CMakeLists.txt
+++ b/Userland/Services/LoginServer/CMakeLists.txt
@@ -9,6 +9,7 @@ compile_gml(LoginWindow.gml LoginWindowGML.h login_window_gml)
 set(SOURCES
     LoginWindowGML.h
     LoginWindow.cpp
+    ../Taskbar/ShutdownDialog.cpp
     main.cpp
 )
```
The only other option apart from that which I'm seeing would be to make a separate LibShutdownDialog, but that feels like it would be way overkill.

Reply:
What about making ShutdownDialog a separate program?

Reply:
That feels a bit overkill to me as well. IMO it's completely fine as nothing more than a class, it's just a matter of where to put it. I'd say keeping it in Taskbar is probably fine, but maybe one of the maintainers knows a better place.

Reply:
I don't think it's overkill, the About system box is its own program as well, plus you generally don't see a program pulling in a file from a different one.